test/index.test.js

```diff
@@ -13,7 +13,8 @@ describe('Mixpanel', function() {
     token: 'x',
     cookieName: 'y',
     crossSubdomainCookie: true,
-    secureCookie: true
+    secureCookie: true,
+    persistence: 'localStorage'
```

Comment:
Looks like they removed get_config() from their mixpanel object...

I wrote this test and it passes but it's super brittle, should we include this test or leave as is
'''
      it('should switch to localStorage persistence if specified', function() {
        mixpanel.options.persistence = 'localStorage';
        analytics.initialize();
        analytics.page();
        analytics.called(mixpanel.load);
        analytics.assert(window.mixpanel._i[0][1].persistence === 'localStorage');
      });
'''